rezerwacje-front/src/App.jsx

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import { Switch, Redirect, Route } from 'react-router-dom';
+import { ThemeProvider } from '@material-ui/styles';
+
+import theme from './theme';
+import './App.scss';
+
+import { Main as MainLayout } from './layouts';
+import Routes from './common'
+
+const App = () => {
+
+  const MainRoutes = () => {
+    return (
+      <Switch>
+        {
+          Routes.map(route => 
+            <Route 
+              key={route.title} 
+              exact 
+              path={route.href} 
+              render={() => route.view}
+            />)
+        }
+      </Switch>
+    );
+  };
```

Comment:
- [ ] By embedding I meant taking what is returned by the component and use it instead of the component